google/cloud/apigee/registry/applications/v1alpha1/registry_manifest.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryManifestProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// Manifest represents the dependency relation between resources, which 
```

Comment:
minor tweak, could we say this? "Manifest represents the dependencies of a list of generated resources that are stored in a registry and is used by the controller to keep these resources up-to-date."

Reply:
Done